zetaclient/btc_signer.go

```diff
@@ -215,7 +210,8 @@ func (signer *BTCSigner) TryProcessOutTx(send *types.CrossChainTx, outTxMan *Out
 	myid := zetaBridge.keys.GetAddress()
 	// Early return if the send is already processed
 	// FIXME: handle revert case
-	included, confirmed, _ := btcClient.IsSendOutTxProcessed(send.Index, int(send.GetCurrentOutTxParam().OutboundTxTssNonce), common.CoinType_Gas, logger)
+	outboundTxTssNonce := send.GetCurrentOutTxParam().OutboundTxTssNonce
+	included, confirmed, _ := btcClient.IsSendOutTxProcessed(send.Index, int(outboundTxTssNonce), common.CoinType_Gas, logger)
```

Comment:
Shouldn't we log the returned error if non-nil?

Reply:
I would actually remove the error from return list but let's keep it as is for now as the function take care of the error contained (it knows all potential errors and print logs) and returns ONLY nil.